server/consumer.go

```diff
@@ -48,7 +48,9 @@ type ConsumerInfo struct {
 }
 
 type ConsumerConfig struct {
+	// Durable is deprecated. All consumers will have names. picked by clients.
 	Durable         string          `json:"durable_name,omitempty"`
+	Name            string          `json:"name,omitempty"`
```

Comment:
Why `omitempty` if `All consumers will have names`?

Reply:
It's good hygiene for older consumers and clients since it does not introduce this on the wire in those situations.